src/theme/components/checkbox.ts

```diff
@@ -0,0 +1,79 @@
+import { checkboxAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers, defineStyle } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(checkboxAnatomy.keys);
+
+const baseStyleControl = defineStyle(() => {
+  return {
+    borderRadius: "2px", // change the border radius of the control
+    border: "2px solid",
+    borderColor: "gray.700",
+    flexShrink: 0,
+    backgroundColor: "white",
+    color: "white",
+    gap: "10px",
+    _checked: {
+      padding: "2px",
+      borderColor: "primary.500",
+      backgroundColor: "primary.500",
+    },
+    _hover: {
+      padding: "2px",
+      borderColor: "brand.800",
+      backgroundColor: "brand.800",
+      color: "white",
+    },
+    _disabled: {
+      backgroundColor: "gray.300",
+      borderColor: "gray.400",
+      color: "gray.400",
+    },
+  };
+});
+
+const baseStyleContainer = defineStyle({
+  // _disabled: { cursor: "not-allowed" },
```

Comment:
As a rule of thumb, let's not commit commented out code. It's not obvious to our future-selves why this is here. If you don't need to set any styles here just do `const baseStyleContainer = defineStyle({});`